axelor-stock/src/main/java/com/axelor/apps/stock/service/StockCorrectionServiceImpl.java

```diff
@@ -190,19 +208,16 @@ public StockMove generateStockMove(StockCorrection stockCorrection) throws Axelo
   public StockMove createStockMoveHeader(
       Company company, StockLocation fromStockLocation, StockLocation toStockLocation)
       throws AxelorException {
-    StockMove stockMove =
-        Beans.get(StockMoveService.class)
-            .createStockMove(
-                null,
-                null,
-                company,
-                fromStockLocation,
-                toStockLocation,
-                null,
-                null,
-                null,
-                StockMoveRepository.TYPE_INTERNAL);
-    return stockMove;
+    return stockMoveService.createStockMove(
+        null,
+        null,
+        company,
+        fromStockLocation,
+        toStockLocation,
+        null,
+        null,
+        null,
+        StockMoveRepository.TYPE_INTERNAL);
   }
```

Comment:
this method should at least not be public, please move it to protected.
If it is called only one time, it is better to remove this method and directly call the stock move service